app/models/friending.rb

```diff
@@ -0,0 +1,6 @@
+class Friending < ApplicationRecord
+
+  belongs_to :friend_initiator, foreign_key: :friender_id, class_name: "User"
+  belongs_to :friend_recipient, foreign_key: :friended_id, class_name: "User"
+
```

Comment:
I think you can add a constraint here to not allow someone from friending themselves